chromadb/test/property/test_embeddings.py

```diff
@@ -90,11 +91,19 @@ def initialize(self, collection: strategies.Collection):
             ids=[], metadatas=[], documents=[], embeddings=[]
         )
 
-    @rule(target=embedding_ids, record_set=strategies.recordsets(collection_st))
-    def add_embeddings(self, record_set: strategies.RecordSet) -> MultipleResults[ID]:
+    @rule(
+        target=embedding_ids,
+        record_set=strategies.recordsets(collection_st),
+        should_compact=st.booleans(),
+    )
+    def add_embeddings(
+        self, record_set: strategies.RecordSet, should_compact: bool
+    ) -> MultipleResults[ID]:
         trace("add_embeddings")
         self.on_state_change(EmbeddingStateMachineStates.add_embeddings)
 
+        initial_version = self.collection.get_model()["version"]
```

Comment:
should this be inside the if not NOT_CLUSTER_ONLY block below to avoid the same race fixed in test_add?

Reply:
Changed the approach so this comment is moot now